jujugui/static/gui/src/app/components/budget-table/row/test-row.js

```diff
@@ -278,9 +285,27 @@ describe('BudgetTableRow', function() {
       acl,
       plansEditable: true
     });
-    assert.equal(wrapper.find('Button').at(0).prop('disabled'), true);
-    assert.equal(wrapper.find('Button').at(1).prop('disabled'), true);
-    assert.equal(wrapper.find('Button').at(2).prop('disabled'), true);
+    assert.equal(
```

Comment:
In this case you actually want them in a single line (like I thought it would have done) as it reads from left to right.

Reply:
Here I agree; this reads oddly to me.